sample-functions/SentimentAnalysis/handler.py

```diff
@@ -1,17 +1,31 @@
 import sys
+import json
 from textblob import TextBlob
 
+reload(sys)
```

Comment:
do you need a one liner in the commit message? or so you want it in a code comment?